dist/output.css

```diff
@@ -4207,7 +4273,7 @@ textarea.form-control.is-invalid {
 html,
 body {
```

Comment:
Background color should be different from the inputs. You can get the background color hex from the figma file.